content/chainguard/chainguard-images/features/ca-docs/custom-assembly.md

```diff
@@ -21,16 +21,18 @@ toc: true
 
 Chainguard has created Custom Assembly, a tool that allows users to create customized container images with extra packages added. This enables customers to reduce their risk exposure by creating container images that are tailored to their internal organization and application requirements while still having few-to-zero CVEs.
 
-This guide outlines how to build customized Chainguard Containers using Custom Assembly in the Chainguard Console. It includes a brief overview of how Custom Assembly works, as well as its limitations.
+This overview of Custom Assembly outlines how it works, its limitations, and how you can use container images customized with Custom Assembly. For a more hands-on tutorial on using Custom Assembly, Chainguard Academy currently has documentation for the following methods of managing the tool:
 
-> **NOTE**: The Custom Assembly tool is currently in its beta phase and it is likely to go through changes before it becomes generally available.
+* [Using the Chainguard console](/chainguard/chainguard-images/features/ca-docs/custom-assembly-console)
+* [Using `chainctl`, Chainguard's command-line interface tool](/chainguard/chainguard-images/features/ca-docs/custom-assembly-chainctl/)
+* [Using Chainguard's API](/chainguard/chainguard-images/features/ca-docs/custom-assembly-api-demo/)
 
 
 ## About Custom Assembly
 
-Custom Assembly is only available to customers that have access to Production Chainguard Containers. Additionally, your account team must enable Custom Assembly before you will be able to begin using it. Contact your account team directly to start the process.
+Custom Assembly is only available to customers that have access to Production Chainguard Containers. Customers with access can use Custom Assembly to customize any container image they have access to.
 
-When you enable the Custom Assembly tool for your organization, you must select at least one of Chainguard's application images to serve as the source for your customized container image. For example, if you want to build a custom base for a Python application, you would likely elect to use the [Python Chainguard Container](https://images.chainguard.dev/directory/image/python/versions) as the source for your customized image.
+Say, for example, you want to build a custom base for a Python application. In this case, you would likely choose to use the [Python Chainguard Container](https://images.chainguard.dev/directory/image/python/versions) as the source for your customized image.
```

Comment:
Missing some context that was here before. Maybe change it to something like "When building a custom assembly image, you will need to choose an appropriate base image. For example, you may choose the Python image as the base for a Python application."